python/sglang/srt/server_args.py

```diff
@@ -96,6 +97,7 @@ class ServerArgs:
     api_key: Optional[str] = None
     file_storage_pth: str = "sglang_storage"
     enable_cache_report: bool = False
+    reasoning_parser: Optional[str] = None
```

Comment:
To remain compatible with clients which haven't implemented support for DeepSeeks default behavior (`separate_reasoning=True`) Microsoft is serving DeepSeek-R1 with `separate_reasoning=False` as the default.

Can we add an argument to change the default for `separate_reasoning`? Or an environment variable? Either way this is crucial functionality for us.

Reply:
Changes proposed in #4001